src/controllers/health/index.ts

```diff
@@ -1,22 +1,46 @@
-import { Controller, Get, Route, Request } from 'tsoa'
-import { Logger } from 'pino'
+import { Controller, Get, Route, SuccessResponse } from 'tsoa'
 
-import * as Ex from 'express'
 import type { Health } from '../../models'
 import { logger } from '../../lib/logger'
+import { startStatusHandlers } from '../../lib/ServiceWatcher/index'
+import { serviceState } from '../../lib/util/statusPoll'
+
 @Route('health')
 export class health extends Controller {
   constructor() {
     super()
   }
 
+  @SuccessResponse(200)
   @Get('/')
-  public async get(@Request() req: Ex.Request & { log: Logger; req_id: string }): Promise<Health> {
+  public async get(): Promise<Health> {
     logger.debug({ msg: 'new request received', controller: '/health' })
 
-    return Promise.resolve({
-      message: 'success',
-      req_id: req.req_id,
-    })
+    const serviceStatusStrings = {
+      [serviceState.UP]: 'ok',
+      [serviceState.DOWN]: 'down',
+      [serviceState.ERROR]: 'error',
+    }
+
+    const statusHandler = await startStatusHandlers()
+    const status = statusHandler.status
```

Comment:
in `dscp-api` the `/health` endpoint returns a code based on `serviceState.UP ? 200 : 503`. In this repo with `tsoa` you'll need to throw a custom `ServiceUnavailable` error like how we throw `NotFound` and `BadRequest` in other controllers. It'll be a bit different to the `NotFound/BadRequest` errors because we send more than just a `message` in the body, I expect you'll need some new code in the error handler func to make it construct the correct response, like how we do for `422`s https://github.com/digicatapult/dscp-matchmaker-api/blob/main/src/lib/error-handler/index.ts#L62-L70